### ISSUE TITLE
Missing "options" property on <select> elements

### ISSUE THREAD

**Issue by anatolsommer** [capricorn86/happy-dom#428]
The `options` property on `<select>` elements is missing leading to problems when using happy-dom in vitest (and probably with Vue in general), see https://github.com/vitest-dev/vitest/issues/1013 .

https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/options

**Comment by johny** [comment]
Yep, this leads to problem when testing `<select>` components, the test fail with: `TypeError: Cannot read properties of undefined (reading 'length')` message.

**Comment by sbayd** [comment]
+1

**Comment by capricorn86** [comment]
Thanks to @fatihcure-jf we now have support for HTMLOptGroupElement, HTMLOptGroupElement and HTMLSelectElement.

You can read more about the release here:
https://github.com/capricorn86/happy-dom/releases/tag/v5.1.0

**Comment by anatolsommer** [comment]
Works now, https://github.com/vitest-dev/vitest/issues/1013 closed. Thank you!
